README.md

```diff
@@ -1,7 +1,9 @@
 [![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+[![Build Status](https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core)
+
 # Google App Engine Plugins Core Library
 
-This is the client Java library to manage App Engine Java applications. It should be used by any application that performs App Engine Java application management. For example, the Maven, Gradle and Eclipse App Engine plugins, custom user tools, etc.
+This is the client Java library to manage App Engine Java applications. It should be used by any Java application that performs App Engine Java application management. For example, the Maven, Gradle and Eclipse App Engine plugins, custom user tools, etc.
```

Comment:
If external users choose to use this library we certainly can't stop them.  But we want to disabuse them of the notion that this is an official supported Google admin API for App Engine.  The naming of the repo was designed to do that, but this wording isn't.  We just need to fix it.